onnxruntime/test/testdata/kernel_def_hashes/onnx.cpu.json

```diff
@@ -1463,6 +1463,30 @@
         "OneHot ai.onnx CPUExecutionProvider",
         15162687909384154912
     ],
+    [
+        "Optional ai.onnx CPUExecutionProvider",
+        4007199385789893408
+    ],
+    [
+        "OptionalGetElement ai.onnx CPUExecutionProvider",
+        8727767224223660008
+    ],
+    [
+        "OptionalHasElement ai.onnx CPUExecutionProvider",
+        103583056104706000
+    ],
+    [
+        "Identity ai.onnx CPUExecutionProvider",
+        17661628575887109792
+    ],
+    [
+        "If ai.onnx CPUExecutionProvider",
+        14605093928914589528
+    ],
+    [
+        "Loop ai.onnx CPUExecutionProvider",
+        3437102755395866648
+    ],
     [
```

Comment:
nit: can we keep the entries alphabetically sorted?

Reply:
Kept it sorted